test/unit/lib/static/components/bottom-progress-bar/index.jsx

```diff
@@ -30,7 +30,7 @@ describe('<BottomProgressBar/> component', () => {
                 visibleRootSuiteIds: ['id1', 'id2', 'id3', 'id4', 'id5']
             });
 
-            assert.equal(component.find('.bottom-progress-bar__progress-bar').prop('style').width, '0%');
+            assert.equal(component.container.querySelector('.bottom-progress-bar__progress-bar').style.width, '0%');
```

Comment:
Why do we need to access `container` property? Can't we call `component.getByClassName` or something like that?